dateutil/parser/_parser.py

```diff
@@ -571,6 +571,74 @@ def resolve_ymd(self, yearfirst, dayfirst):
         return year, month, day
 
 
+def next_jump(tokens, idx, skip_comma=True):
+    """
+    Get the next sibling and its index, skipping over whitespace
+    and (if specified) commas.
+
+    Parameters
+    ----------
+    tokens : list[str]
+    idx : int
+    skip_comma : bool, default True
+
+    Returns
+    -------
+    sib : str or None
+    sib_idx : int or None
+    """
+    if idx == len(tokens) - 1:
+        (sib, sib_idx) = (None, None)
+    else:
+        sib_idx = idx + 1
+        sib = tokens[sib_idx]
+        if sib.isspace():
+            (sib, sib_idx) = next_jump(tokens, idx + 1)
+        elif (skip_comma and sib == ',' and idx + 3 < len(tokens) and
+              tokens[idx + 2].isspace()):
+            # e.g. if tokens[idx] is the "30" in "Dec 30, 2016" we want to
+            #  return the "2016"
+            sib_idx = idx + 3
+            sib = tokens[sib_idx]
+
+    return sib, sib_idx
+
+
+def prev_jump(tokens, idx, skip_comma=True):
```

Comment:
It doesn't seem like you're actually using this function anywhere, other than in the function itself. Is it necessary to keep it?

Reply:
We don't need it at the moment, but I thought I'd port it since it goes with next_jump.  I'll remove it now and can re-introduce it if/when it becomes necessary.